Source/WebCore/dom/TextDecoder.cpp

```diff
@@ -81,7 +81,7 @@ ExceptionOr<String> TextDecoder::decode(std::optional<BufferSource::VariantType>
 
 String TextDecoder::encoding() const
 {
-    return makeString(asASCIILowercase(m_textEncoding.name()));
+    return makeString(asASCIILowercase(StringView { m_textEncoding.name() }));
```

Comment:
These are all guaranteed to be ASCII. I wonder if there’s a way we could do this without calling `StringView::fromLatin1`, but maybe that’s OK.